v1/content.yaml

```diff
@@ -161,7 +163,7 @@ paths:
           type: string
       produces:
         - text/html; charset=utf-8; profile="https://www.mediawiki.org/wiki/Specs/HTML/1.6.0"
-        - application/json
+        - application/problem+json
```

Comment:
This is incorrect. this endpoint can return actual json in case section ids are provided

Reply:
Ok, brought both of these back. Thanks!